SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -670,6 +670,11 @@ void USpatialNetDriver::OnLevelAddedToWorld(ULevel* LoadedLevel, UWorld* OwningW
 		return;
 	}
 
+	if (!IsServer())
```

Comment:
will remove this before merging, but you can't do any tests without it

Reply:
it's nice to see that CI passes regardless of whether or not this is here though

Reply:
Get this outta here

Reply:
done